react/__test__/index.test.tsx

```diff
@@ -116,19 +116,6 @@ describe('Rich text component', () => {
     expect(component.asFragment()).toMatchSnapshot()
   })
 
-  it('should render with image block class props', () => {
```

Comment:
This test doesn't make sense. blockClass is rendered through css-handles with render-runtime magic